Formula/libsais.rb

```diff
@@ -0,0 +1,43 @@
+class Libsais < Formula
+  desc "Fast linear time suffix array, lcp array and bwt construction"
+  homepage "https://github.com/IlyaGrebnov/libsais"
+  url "https://github.com/IlyaGrebnov/libsais.git",
+    tag:      "v2.7.1",
+    revision: "c0b2bba165ae759fcc034d8f22d973cea8d7329c"
+  license "Apache-2.0"
+  head "https://github.com/IlyaGrebnov/libsais.git", branch: "master"
+
+  depends_on "gcc" => :build
+  depends_on "make" => :build
```

Comment:
```suggestion
```

Reply:
Why "make" should not be listed as a dependency?